### DISCUSSION TITLE
Is it possible to export a selected portion as PNG/PDF/SVG

### DISCUSSION THREAD

**Discussion by GoBigorGoHome** [otfried/ipe#471] [Q&A]
As described in title.

**Comment by otfried** [comment]
No, there is no direct way to do this.

If it's a simple drawing, you can make a copy, group everything together, and set a clip-path on the group to achieve what you want (https://otfried.github.io/ipe/30_objects.html#clipping).

For anything else, an easy way to do it is from Latex using `\includegraphics` using the `clip` and the `trim` or `viewport` arguments.
